OutOfSchool/OutOfSchool.DataAccess/Repository/ChangesLogRepository.cs

```diff
@@ -68,6 +71,27 @@ public ChangesLogRepository(OutOfSchoolDbContext dbContext)
         return result;
     }
 
+    public async Task<ChangesLog> AddCreatingOfEntityToChangesLog<TEntity>(
```

Comment:
(Code style, performance) `async`/`await` seems to be excessive for this method, as have only one `await` in method body that is immediately returned. Please refactor

Reply:
Done. Thanks.